cmd/server/main.go

```diff
@@ -304,6 +305,18 @@ func realMain(ctx context.Context) error {
 		realmSub.Handle("/keys/activate", realmKeysController.HandleActivate()).Methods("POST")
 	}
 
+	// jwks
+	{
+		jwksSub := r.PathPrefix("/jwks").Subrouter()
+		jwksSub.Use(rateLimit)
+
+		jwksController, err := jwks.New(ctx, db, cacher, h)
+		if err != nil {
+			return fmt.Errorf("failed to create jwks controller: %w", err)
+		}
+		jwksSub.Handle("/{realm}/{keyid}", jwksController.HandleIndex()).Methods("GET")
```

Comment:
The key id doesn't need to be in the URL - a single endpoint should list all non-destroyed keys for a realm.
That way the exposure-notificiations-server has a single document to point to.

Sorry if our chat conversation steered you wrong here.

Reply:
It did. I'll fix and reupload.

Reply:
Ok, this might be closer to what you had in mind. LMK.